web/src/Routes.tsx

```diff
@@ -10,13 +10,21 @@
 import { Router, Route, Set } from '@redwoodjs/router'
 
 import NewsLayout from 'src/layouts/NewsLayout'
+import SignInSignUpLayout from 'src/layouts/SignInSignUpLayout'
+
+import { useAuth } from './auth'
 
 const Routes = () => {
   return (
-    <Router>
+    <Router useAuth={useAuth}>
```

Comment:
need a comment for what useAuth does. who is authorized to see these pages.

Reply:
useAuth is the use hook that holds the state of whos currently logged in and manages logging out users from the database